### PR TITLE
update scientifc notation

### PR DESCRIPTION
**Pull request by jude-zhu**

### CODE REVIEW

File: docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md [hunk 1]
```diff
@@ -14,16 +14,11 @@
...
-## 示例
-
-以下为几个例子：
+Nebula Graph 中，支持科学计数法表示 Double 类型。例如：
 
 ```ngql
-1, -5, +10000100000
--2.3, +1.00000000000
-1.2E3
+> CREATE TAG test1(price DOUBLE)
```
Comment (Amber1990Zhang) [inline docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md:22]: nebula> CREATE TAG test1(price DOUBLE);

File: docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md [hunk 2]
```diff
@@ -14,16 +14,11 @@
 
 双浮点数上限和下限分为别 `-1.79769e+308` 和 `1.79769e+308`。
 
-## 科学计数法
+**科学计数法**
```
Comment (Amber1990Zhang) [inline docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md:17]: I think this violates the markdown lint rules. Please change it back to header.
image

File: docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md [hunk 3]
```diff
@@ -14,16 +14,11 @@
...
 
-科学计数法是指以尾数和指数表示的数字。尾数或指数中的一个或两个都可以带负号。例如：`1.2E3`、`1.2E-3`、`-1.2E3`、`-1.2E-3`。
-
-## 示例
-
-以下为几个例子：
+Nebula Graph 中，支持科学计数法表示 Double 类型。例如：
```
Comment (Amber1990Zhang) [inline docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md:19]: 支持用科学计数法表示 Double 类型

File: docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md (hunk 2 above)
Comment (whitewum) [reply docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md:17]: I agree

File: docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md (hunk 3 above)
Comment (whitewum) [inline docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md:19]: 左边的句子是抄MySQL的

File: docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md (hunk 1 above)
Comment (whitewum) [reply docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md:22]: I agree

File: docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md [hunk 4]
```diff
@@ -14,16 +14,11 @@
...
-## 示例
-
-以下为几个例子：
+Nebula Graph 中，支持用科学计数法表示 Double 类型。例如：
 
 ```ngql
-1, -5, +10000100000
--2.3, +1.00000000000
-1.2E3
+nebula> CREATE TAG test1(price DOUBLE)
```
Comment (Amber1990Zhang) [inline docs/manual-CN/2.query-language/3.language-structure/literal-values/numeric-literals.md:22]: please add a ; at the end of each query

**Pull request merged**